daprdocs/content/en/docs/developing-applications/_index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Developing applications with Dapr"
-shortTitle: "Developing"
+linkTitle: "Developing applications"
```

Comment:
Looking at the staging site and the lowercase "applications" doesn't look right to me in context with the rest of the titles. I think we should either make it "Developing Applications" or switch "Getting Started" to "Getting started".

Reply:
I believe we discussed this, I kept it consistent in my section, only capitals in start and with names. 2 exceptions we can discuss are "How-To" and "Pub/Sub" which I kept like so but also okay with changing.
To the point of this comment, will change getting started (I didn't really touch anything outside this section but will add this to the PR)